pinot-segment-local/src/test/resources/data/newColumnsSchema3.json

```diff
@@ -34,10 +34,6 @@
       "name": "newBooleanSVDimension",
       "defaultNullValue": "false"
     },
-    {
```

Comment:
Why removing this field?

Reply:
to test the logic to remove auto-gen'ed default column.